client/my-sites/email/email-providers-in-depth-comparison/comparison-table/index.tsx

```diff
@@ -15,7 +15,7 @@ type ComparisonTableProps = {
 };
 
 const ComparisonTable: FunctionComponent< ComparisonTableProps > = () => {
```

Comment:
Non-blocking nit: It seems `React.FunctionComponent` is no longer encouraged ([source](https://github.com/facebook/create-react-app/pull/8177)). It may be better to define a return type of `ReactElement | null` for the function instead.

Reply:
I will address that in a future iteration 👌.

Reply:
This will be addressed in https://github.com/Automattic/wp-calypso/pull/59775.